packages/fluentui/react-builder/src/components/Toolbar.tsx

```diff
@@ -41,68 +41,138 @@ export const Toolbar: React.FunctionComponent<ToolbarProps> = ({
       display: 'flex',
       padding: '0 1rem',
       alignItems: 'center',
-      boxShadow: '0 1px 3px rgba(0, 0, 0, 0.25)',
+      borderBottom: '1px solid #E1DFDD', //TODO: replace with `Default Border 2` color scheme token
+      background: '#FAF9F8', //TODO: replace with `Default Background 1` color scheme token
       ...style,
     }}
   >
     <Image
-      styles={{ height: '1.5rem', marginRight: '0.25rem' }}
+      styles={{ height: '1.5rem' }}
       src="https://fabricweb.azureedge.net/fabric-website/assets/images/fluent-ui-logo.png"
     />
-    <div style={{ position: 'relative', width: '8em', fontSize: '18px', lineHeight: 1 }}>
-      FluentUI
-      <div style={{ position: 'absolute', fontSize: '11px', opacity: 0.625 }}>Builder</div>
+    <div style={{ position: 'relative', padding: '0 .8rem', fontSize: '14px', lineHeight: 1, fontWeight: 'bold' }}>
+      FluentUI <span style={{ fontWeight: 'normal' }}>Builder</span>
     </div>
-    <div>
-      <strong>Mode:</strong>
-      &emsp;
-      <RadioGroup
-        style={{ display: 'inline-block' }}
-        checkedValue={mode}
-        onCheckedValueChange={(e, data: RadioGroupItemProps & { value: DesignerMode }) => {
-          onModeChange(data.value);
-        }}
+    <div style={{ display: 'flex', alignItems: 'center', marginLeft: 'auto' }}>
+      <div>
+        <MenuButton
+          trigger={<Button text iconOnly content={mode} aria-label="Select mode" />}
```

Comment:
What is the best way to do this so that I can swap the button text with the content from the selected dropdown item?

Reply:
I think this is the right way - to use state

Reply:
Thanks @jurokapsiar - I added in a replace to convert the first letter to uppercase